{{cookiecutter.hyphenated}}/README.md

```diff
@@ -0,0 +1,41 @@
+# {{cookiecutter.github_username}}/{{cookiecutter.hyphenated}}
+
+<!-- TODO: Update these based on your pipeline's supported platforms -->
```

Comment:
Ah so some of the `TODO` statements are still here - just curious about which ones remain or which ones we want the end user to fill out since I do like the `TODO` functionality in VS Code